excel-importer/src/test/java/uk/gov/hmcts/ccd/definition/store/excel/service/ImportServiceImplTest.java

```diff
@@ -256,39 +272,14 @@ public void shouldImportDefinition() throws Exception {
 
     @Test
     public void shouldGetUserDetails() {
```

Comment:
I think this test is not needed anymore. What do we test in here? That the mocking works? I would add in shouldImportDefinition() 
verify(idamProfileService).getLoggedInUserDetails();
Then inline the call and remove @VisibleForTesting annotation.